docs/topics/Permissions.md

```diff
@@ -80,6 +80,7 @@ Below is a table of all current permissions, their integer values in hexadecimal
 | USE_EXTERNAL_SOUNDS                    | `0x0000200000000000` `(1 << 45)` | Allows the usage of custom soundboard sounds from other servers                                                                                                                                                                                            | V            |
 | SEND_VOICE_MESSAGES                    | `0x0000400000000000` `(1 << 46)` | Allows sending voice messages                                                                                                                                                                                                                              | T, V, S      |
 | SEND_POLLS                             | `0x0002000000000000` `(1 << 49)` | Allows sending polls                                                                                                                                                                                                                                       | T, V, S      |
+| USE_EXTERNAL_APPS                      | `0x0004000000000000` `(1 << 50)` | Allows using user installed apps not installed to the server. When disabled, users will still be allowed to use their apps but the responses will be ephemeral.                                                                                            | T, V, S      |
```

Comment:
```suggestion
| USE_EXTERNAL_APPS                      | `0x0004000000000000` `(1 << 50)` | Allows user-installed apps not installed to the server to send public responses. When disabled, users will still be allowed to use their apps but the responses will be ephemeral.                                                                                            | T, V, S      |
```

Reply:
maybe something like this that focuses on the responses? because right now the first sentence almost feels misleading in context.

Reply:
Made an update, I really like your suggestion.